lazy_tensor_core/test/cpp/test_aten_ltc_ts_tensor.cpp

```diff
@@ -179,6 +179,18 @@ TEST_F(AtenLtcTsTensorTest, TestAdd) {
   });
 }
 
+TEST_F(AtenLtcTsTensorTest, TestAddHalf) {
```

Comment:
do we need any special tests for mixed precision?  (well, type-promotion, really.). I.e. can lazy tensor smoothly handle adding f16 + f32?

Reply:
@wconstab Pretty sure it does, yes, we insert explicit numeric casts.